src/engine/output-results.ts

```diff
@@ -217,6 +230,7 @@ function nextState(
         status: 'status' in resource ? resource.status : 'unknown'
       };
     } else {
+      // TODO: possibly add search for a note here
```

Comment:
Yeah I think you will have to look for Declines here

Reply:
Added search for notes on decline. Now reads note and marks the node as in progress but declined
![Screen Shot 2020-03-05 at 2 44 11 PM](https://user-images.githubusercontent.com/53485517/76019485-c8f64980-5eef-11ea-9e55-ebe8b45d5966.png)